### ISSUE TITLE
upgrade to librabbitmq-c v0.7.0?

### ISSUE THREAD

**Issue by markwellis** [net-amqp-rabbitmq/net-amqp-rabbitmq#59]
https://github.com/alanxz/rabbitmq-c/releases/tag/v0.7.0

according to the release notes, there's plenty of internal socket changes, so it wants testing properly, probably best to do it after #53 and #11 / #27 when the test suite is in better shape

**Comment by manchicken** [comment]
Yay

**Comment by manchicken** [comment]
I'm going to v0.7.1 in a new pull request.

**Comment by markwellis** [comment]
this is released in version 1.1, which will be on cpan shortly
